### DISCUSSION TITLE
lndmobile: (Trying to figure it all out)  proto: cannot parse invalid wire-format data

### DISCUSSION THREAD

**Discussion by pauldando** [lightningnetwork/lnd#7569] [Troubleshooting]
Hi all,
Have dipped my toes into trying to get the Daemon running, have managed to compile everything, but now sit with two issues, namely:
1. On calling lndmobile.start I get  unknown flag `/data/user/0/com.example.hatepurehate/files/.lnd' with my cmd being file.getAbsoluteFile() + "/.lnd"+" --bitcoin.active --bitcoin.testnet", but when I call it a second time I get go.Universe$proxyerror: intercept already started.

2. Calling lndmobile.getChainInfo yields this confusing error : proto: cannot parse invalid wire-format data.

Any help would be appreciated.

**Comment by guggero** [comment]
The "unknown flag" error, what does it look like exactly? Sounds to me like there is additional info in the full error message that might help to resolve the first problem.

The second issue seems to be that you aren't passing in a protobuf serialized message into `getChainInfo()`. You need to create a protobuf message and serialize it.

Maybe you want to take a look at https://github.com/hsjoberg/blixt-wallet, which uses the `lndmobile` code for Android and iOS, you'll probably be able to pull some examples from there.
